pynestml/codegeneration/builder.py

```diff
@@ -34,7 +39,15 @@
 class Builder(WithOptions, metaclass=ABCMeta):
     r"""Compile, build and install the code for a given target platform. Runs after the CodeGenerator."""
 
-    def __init__(self, target, options: Optional[Mapping[str, Any]]=None):
+    _redirect_build_output = {
+        "error_location": "stderr",
+        "default": True,
+        "stdout": None,
+        "stderr": subprocess.STDOUT
+
+    }
```

Comment:
As Builder inherits from WithOptions, can you just use the get and set-option methods that already exists? The defaults in this ``_redirect_build_output`` dict should be moved to the ``_default_options``.